docs: add link for serve CLI options (#4611)
webpack/webpack.js.org: src/content/configuration/dev-server.md

T> HTML template is required to serve the bundle, usually it is an `index.html` file. Make sure that script references are added into HTML, webpack-dev-server doesn't inject them automatically.

### Usage via CLI

You can invoke webpack-dev-server via CLI by: 

```bash
npx webpack serve
```

> See a list of available CLI options for serve [here](https://github.com/webpack/webpack-cli/blob/master/SERVE.md)

Reviewer: ```suggestion
List of CLI options for serve is available [here](https://github.com/webpack/webpack-cli/blob/master/SERVE.md)
```

Author: ```suggestion
A list of CLI options for `serve` is available [here](https://github.com/webpack/webpack-cli/blob/master/SERVE-OPTIONS.md)
```
